cluster/router/tag/router_rule.go

```diff
@@ -34,5 +52,58 @@ func getRule(rawRule string) (*RouterRule, error) {
 		return r, err
 	}
 	r.RawRule = rawRule
+	r.init()
 	return r, nil
 }
+
+func (t *RouterRule) init() {
```

Comment:
Why create a new func named init? If you create the func ,pls add the comment for it.